addons/website_sale/models/product_product.py

```diff
@@ -278,7 +278,7 @@ def _prepare_gmc_image_links(self, base_url):
         self.ensure_one()
         return {
             # Don't send any image link if there isn't. Google does not allow placeholder
-            'image_link': urljoin(base_url, self._get_image_1920_url()) if self.image_1920 else '',
+            'image_link': urljoin(base_url, self._get_image_1920_url()) if (self.image_variant_128 or self.product_tmpl_id.image_128) else '',
```

Comment:
You can use `image_128` directly
```suggestion
            'image_link': urljoin(base_url, self._get_image_1920_url()) if self.image_128 else '',
```

Reply:
Hello @lipiraux ,

Thanks for the review.
Suggested changes are done.